test/images/image-util.sh

```diff
@@ -165,7 +165,7 @@ build() {
             if [[ $(id -u) -ne 0 ]]; then
 	            sudo="sudo"
             fi
-            ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset -p yes
+             ${sudo} docker run --rm --privileged tonistiigi/binfmt:latest --install all
```

Comment:
Does this intentionally have leading whitespace?

Reply:
whoops! fixing